gamelauncher.py

```diff
@@ -49,13 +49,36 @@ def parse_args() -> Namespace:  # noqa: D103
 
 
 def _setup_pfx(path: str) -> None:
-    """Create a symlink to the WINE prefix and tracked_files file."""
+    """Create a symlink to the WINE prefix and tracked_files file.
+
+    Also, create a symlink of steamuser to the Unix username
+    """
+    username: str = Path().home().name
```

Comment:
The lack of guarantee is that the home directory might not match the user name. You can have your HOME directory with a different stem than your username. It's is unlikely generally but there is nothing blocking it from happening, see `useradd --home-dir`.

As far as I can tell, because Wine source is not easy to navigate, the username is assigned [here](https://github.com/wine-mirror/wine/blob/master/dlls/ntdll/unix/loader.c#L384). In my opinion, inferring it from `Path.home` is the least accurate one. Replicating Wine's way of getting the user id, we can either base it on the `USER` env variable, or the password database through the [`pwd` module](https://docs.python.org/3/library/pwd.html), for example `pwd.getpwuid(os.getuid())[0]`.

Or is a more simple way, trust python and get it with [`os.getlogin()`](https://docs.python.org/3/library/os.html#os.getlogin)